docs/getting-started/github-apps/github-apps.rst

```diff
@@ -6,6 +6,12 @@ Automatic Syncing with GitHub Apps and .dockstore.yml
 
 Overview
 --------
+Summary of GitHub App usage:
+
+- Install Dockstore GitHub App
+- Push a new commit with a .dockstore.yml file present
+- Wait a while for Dockstore to process the new version
```

Comment:
Maybe "Wait up to 5 minutes"? In my experience, it takes about a minute and half, so that should give us buffer. Or 10, to be extra safe.